modules/store/src/meta-reducers/immutability_reducer.ts

```diff
@@ -14,33 +14,32 @@ export function immutabilityCheckMetaReducer(
   };
 }
 
-function freeze(target: any) {
-  Object.freeze(target);
+export function freeze(target: any) {
+  if (!Object.isFrozen(target)) {
```

Comment:
Can we invert this check to have an indention less?

```suggestion
  if (Object.isFrozen(target)) {
    return target;
  }
```

Reply:
Updated!